src/components/titles/index.js

```diff
@@ -75,6 +75,24 @@ function draw(gd, titleClass, options) {
     var fontLineposition = font.lineposition;
     var fontShadow = font.shadow;
 
+    // Get subtitle properties
+    var subtitleProp = options.subtitlePropName;
+    var subtitleEnabled = !!(subtitleProp);
+    var subtitlePlaceholder = options.subtitlePlaceholder;
+
+    var subtitle = cont.title.subtitle;
```

Comment:
```suggestion
    var subtitle = (cont.title || {}).subtitle;
```
So that `polar_blank`, etc. would render.